codex-ui/src/vue/components/note-header/NoteHeader.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div
+    :class="$style['note-header']"
+    :style="{ '--opacity': props.opacity }"
+  >
+    <div :class="$style['note-header__left']">
+      {{ lastEdit }}
+    </div>
+    <div :class="$style['note-header__right']">
+      <Button
+        icon="Plus"
+        :class="$style['note-header__button']"
+      />
+      <Button
+        icon="EtcHorisontal"
+        :class="$style['note-header__button']"
+      />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+import Button from '../button/Button.vue';
+
+const props = withDefaults(
+  defineProps<{
+    /**
+     * Explanatory caption for the date
+     */
+    description?: string;
```

Comment:
is it used?

Reply:
Yes, here
![image](https://github.com/codex-team/notes.web/assets/94054053/2b6dea76-f2d5-467d-84a3-77cef12df154)

I can remove it and left just static text